lib/schema/export.js

```diff
@@ -102,49 +102,53 @@ function valueToSchemaTypeObject(value) {
   // Note: We currently do not distinguish between a RefValue and other Value
   if (value.identifier.isPrimitive()) {
     // TO CONSIDER: These could all be defined as separate (resusable) types in our types.schema.json.
+    var val = `http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/${value.identifier.name}`;
+    return {'$ref': val};
```

Comment:
There seems to be little value in the separate `val` variable.  I'd suggest making the whole return a one-liner:
```
return {'$ref': `http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/${value.identifier.name}`};
```

If not, then best practice would be to use `const val` rather than `var val`, since `const` is more efficient when you're not going to re-assign the variable.